src/optimization-given.jl

```diff
@@ -83,7 +83,11 @@ function optimization_given(allowgw=false, allowreservoirs=true, demandmodel=not
         gwwo = deserialize(open(cachepath("partialhouse$suffix.jld"), "r"));
         cwro = deserialize(open(cachepath("partialhouse2$suffix.jld"), "r"));
         if allowreservoirs
-            gror = deserialize(open(cachepath("partialhouse-gror$suffix.jld"), "r"));
+	    if isfile(cachepath("partialhouse-gror$suffix.jld"))
+		    gror = deserialize(open(cachepath("partialhouse-gror$suffix.jld"), "r"));
+	    else
+		    gror = grad_reservoir_outflows_captures(m);
+	    end
```

Comment:
Isn't the check for this file above here?  Like with gwwo and cwro.

Reply:
Sometimes, one can perform the optimization without the reservoir and then with when comparing the value of infrastructure. This was a convenient way to not have to regenerate everything. What do you think?

Reply:
Okay, makes sense.